### ISSUE TITLE
Religions that have their holy city captured by another founder disappear from the Religion Overview window

### ISSUE THREAD

**Issue by Edaka** [LoneGazebo/Community-Patch-DLL#1503]
So, Catholicism used to be a pretty significant world religion - adopted in 14 cities, enhanced etc. As soon as I (founder of my own religion) conquer Lisbon, bam - it's as if Catholicism never existed. 

What's also important, it's not only gone from the World Religions tab, it's also not mentioned in the Beliefs tab, so it's not possible its beliefs despite the fact that it's still the majority in 13 cities. Is this intentional?
![20160324232552_1](https://cloud.githubusercontent.com/assets/12978597/14042538/7810f6dc-f283-11e5-825d-fc04eff726a8.jpg)
![20160324234608_1](https://cloud.githubusercontent.com/assets/12978597/14042541/812f76c6-f283-11e5-82b1-553acee85d52.jpg)

**Comment by JFDaly** [comment]
It's because of the way Firaxis coded this display - it iterates over the players that are alive, and not over the religions.

**Comment by Edaka** [comment]
Oh, but I didn't wipe Portugal out. As soon as her holy city was mine, this happened. She's actually alive and well (well, at least her body is).

**Comment by LoneGazebo** [comment]
I haven't edited that lua, and I don't intend to. If someone else, JFD for example, would like to take a look, I'd be thrilled. I hate lua.

G

**Comment by JFDaly** [comment]
Yup, I can take a look.

**Comment by JFDaly** [comment]
Ok, so I've changed how the World Religions and World Beliefs lists are populated, so that they'll register religions belonging to the same player (it might be prudent to update 'TXT_KEY_RO_WR_FOUNDER' to say 'Head' instead of 'Founder'). This fixes the above issue. See here, where I control both the Buddhist and the Catholic Holy City:
![text](http://i.imgur.com/CfT8Hvi.jpg)
![text](http://i.imgur.com/A3UApVg.jpg)

I've marked the changes with JFD:, and commented out the original. The merges were done to the March 23rd version of ReligionOverview.lua in (6b) Community Balance Patch - Compatibility Files (No-EUI) (there are no differences between No-EUI and EUI ReligionOverview, AFAIK). 

Here's [the file](https://www.dropbox.com/s/hqckfybzoqjybny/ReligionOverview.lua?dl=0). Let me know if there are any problems.
